JavaScript/JSON/Types/Instances.hs

```diff
@@ -1058,27 +1058,6 @@ typeMismatch expected actual =
              Bool _   -> "Boolean"
              Null     -> "Null"
 
-realFloatToJSON :: RealFloat a => a -> Value
-realFloatToJSON d
-    | isNaN d || isInfinite d = nullValue
-    | otherwise               = doubleValue (realToFrac d)
-{-# INLINE realFloatToJSON #-}
-
-scientificToNumber :: Scientific -> Number
-scientificToNumber s
-    | e < 0     = D $ Scientific.toRealFloat s
-    | otherwise = I $ c * 10 ^ e
-  where
-    e = Scientific.base10Exponent s
-    c = Scientific.coefficient s
-{-# INLINE scientificToNumber #-}
-
-parseRealFloat :: RealFloat a => String -> Value -> Parser a
-parseRealFloat _        (match -> Number d) = pure $ realToFrac d
-parseRealFloat _        (match -> Null)     = pure (0/0)
-parseRealFloat expected v                   = typeMismatch expected v
-{-# INLINE parseRealFloat #-}
-
```

Comment:
This is not currently used, but is referenced by code that has been commented out.